tasks/pecl.yml

```diff
@@ -21,6 +21,6 @@
   when: php.peclpackages is defined
 
 - name: Enable extension
-  shell: "{% if php_version == 'php5' %}php5enmod{% else %}phpenmod{% endif %} {{ item }}"
+  shell: "phpenmod {{ item }}"
   with_items: php.peclpackages
```

Comment:
@Brunty as of Ansible 2.2 bare variables in conjunction with `with_` keys need to be templated, more [here](https://github.com/ansible/ansible/blob/devel/CHANGELOG.md#removed-deprecated). Meanwhile, would you mind if possible also to update the PR to follow ansible syntax standards so to have templated variable written like `{{ var }}`?